sandboxed_containers/deploying-sandboxed-container-workloads.adoc

```diff
@@ -37,3 +37,8 @@ include::modules/sandboxed-containers-triggering-installation-kata-runtime.adoc[
 include::modules/sandboxed-containers-selecting-nodes.adoc[leveloffset=+3]
 include::modules/sandboxed-containers-scheduling-workloads.adoc[leveloffset=+3]
 include::modules/sandboxed-containers-viewing-workloads-from-cli.adoc[leveloffset=+2]
+
+== Additional resources
+
+* The {sandboxed-containers-operator} is supported in a restricted network environment. For more information, xref:../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks].
+* When using a disconnected cluster on a restricted network, you must xref:../operators/admin/olm-configuring-proxy-support.adoc#olm-configuring-proxy-support[configure proxy support in Operator Lifecycle Manager] to access the Red Hat-provided OperatorHub. Using a proxy allows the cluster to fetch the {sandboxed-containers-operator}.
```

Comment:
```suggestion
* When using a disconnected cluster on a restricted network, you must xref:../operators/admin/olm-configuring-proxy-support.adoc#olm-configuring-proxy-support[configure proxy support in Operator Lifecycle Manager] to access the OperatorHub. Using a proxy allows the cluster to fetch the {sandboxed-containers-operator}.
```

Reply:
Hey @sagidlow - thanks! I copied this content from another location in the help, but I'll make sure to remove it. :) Once I do, do I submit the PR for peer review again?

Reply:
Updated.